trace_test.go

```diff
@@ -226,11 +226,21 @@ func TestResourceDetectionTrace(t *testing.T) {
 		t.Skip("Local runs do not need to test resource detection")
 	case args.Gce != nil:
 		labelCases = append(labelCases,
-			labelExpectation{expectKey: "g.co/r/gce_instance/zone", expectRe: `.*-.*-.*`},
-			labelExpectation{expectKey: "g.co/r/gce_instance/instance_id", expectRe: `.*`},
+			labelExpectation{expectKey: "cloud.provider", expectRe: `gcp`},
+			labelExpectation{expectKey: "cloud.platform", expectRe: `gcp_compute_engine`},
+			labelExpectation{expectKey: "cloud.region", expectRe: `.*-.*`},
+			labelExpectation{expectKey: "cloud.availability_zone", expectRe: `.*-.*-.*`},
+			labelExpectation{expectKey: "host.id", expectRe: `.*`},
 		)
 	case args.Gke != nil:
 		labelCases = append(labelCases,
+			labelExpectation{expectKey: "cloud.provider", expectRe: `gcp`},
+			labelExpectation{expectKey: "cloud.platform", expectRe: `gcp_kubernetes_engine`},
+			labelExpectation{expectKey: "cloud.region", expectRe: `.*-.*`},
+			labelExpectation{expectKey: "k8s.cluster.name", expectRe: `.*`},
+			labelExpectation{expectKey: "k8s.namespace.name", expectRe: `.*`},
+			labelExpectation{expectKey: "k8s.pod.name", expectRe: `.*`},
+			labelExpectation{expectKey: "k8s.container.name", expectRe: `.*`},
 			labelExpectation{expectKey: "g.co/r/k8s_container/location", expectRe: `.*-.*`},
```

Comment:
Maybe just add a comment why we test for g.co/r labels only for gke

Reply:
done